src/app/modules/myinfo/myinfo.util.ts

```diff
@@ -482,16 +482,14 @@ export const getMyInfoAttr = (
  * Helper function to get a MyInfo child's hash key inside an IHashes.
  *
  * @param fieldId The ID of the field the Child response belongs to.
- * @param childIdx The nth child to look for.
  * @returns An IHashes-compatible key.
  */
 export const getMyInfoChildHashKey = (
   fieldId: string,
   childAttr: MyInfoChildAttributes,
-  childIdx: number,
   childName: string,
 ): MyInfoChildKey => {
-  return `${MyInfoAttribute.ChildrenBirthRecords}.${fieldId}.${childAttr}.${childIdx}.${childName}`
```

Comment:
same here; why remove the child idx?

Reply:
re-added [d173907](https://github.com/opengovsg/FormSG/pull/6745/commits/d1739075b59d65998b0b0ea73afa73f0af57b578)